code/game/objects/structures/signs.dm

```diff
@@ -86,6 +86,7 @@
 /obj/structure/sign/warning
 	name = "\improper WARNING"
 	icon_state = "securearea"
+	directional_offset = "{'NORTH':{'y':-32}, 'SOUTH':{'y':32}, 'WEST':{'x':34}, 'EAST':{'x':-34}}"
```

Comment:
Alright, I'll make another PR with that.
But the main reason this was done is because the signs would cover the airlock controls, so I moved them, and then noticed that the auto-offset for them was off because not all signs seems to be oriented the same way on their sprites.

Reply:
https://github.com/NebulaSS13/Nebula/pull/2843